experimental/oak_functions_with_envoy/README.md

```diff
@@ -0,0 +1,80 @@
+# Oak Functions with Envoy Proxy Experiment
+
+This experiment shows how to create a persistent connection to a Docker
+container in Cloud Run. It is done by using
+[TCP tunneling over HTTP](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http/upgrades#tunneling-tcp-over-http)
+in Envoy proxy.
+
+The experiment consists of 2 Envoy proxies: _client side_ and _server side_.
+
+- Client side proxy accepts TCP streams and tunnels them over HTTP streams.
```

Comment:
Also mention that it listens for TCP connections on a local port? Or a Domain Socket?